### PR TITLE
Feature/spack flux

### PR DESCRIPTION
**Pull request by christopherwharrop-noaa**
This PR completes the process of removing the use of conda from Chiltepin.  An installation script to install and build flux and pytest and parsl is added for use on HPC machines.  Also, test configurations for Hera and Hercules are updated to use the official spack-stack installations on those machines.

### CODE REVIEW

**Comment by christopherwharrop-noaa** [comment]
@NaureenBharwaniNOAA - This is ready for you.

File: README.md
Comment (NaureenBharwaniNOAA) [inline README.md:74]: 'needs to only be done once`

File: install/install.sh
Comment (NaureenBharwaniNOAA) [inline install/install.sh:35]: `install ___`? The spack?

**Review by NaureenBharwaniNOAA — DISMISSED** [review]
Just a couple small possible edits.

File: install/install.sh
Comment (christopherwharrop-noaa) [reply install/install.sh]: Thanks for the feedback.  I've pushed some updates.  I'll need you to re-approve if you like the changes.

**Review by NaureenBharwaniNOAA — APPROVED** [review]
Once it passes, approval looks good to me!

**Pull request merged**
